script/deploy-dev.mk

```diff
@@ -37,6 +37,17 @@ deploy-dev-kubeapps:
 		--values ./docs/user/manifests/kubeapps-local-dev-auth-proxy-values.yaml \
 		--values ./docs/user/manifests/kubeapps-local-dev-additional-kind-cluster.yaml
 
+deploy-dev-kubeapps-with-apis:
```

Comment:
Not sure if others would find this useful until we have support in the chart, but I find it really handy to build the image tag, load it to my kind cluster, then just do `make deploy-dev-kubeapps-with-apis` . Obviously, you need to update the image tag, and if you want redis enabled etc.

Reply:
Great, thank you! I ended up creating a separate local values.yaml file and a local script doing: 

```bash
kubectl scale deploy/kubeapps-internal-kubeappsapis -n kubeapps --replicas=0
IMAGE_TAG=dev1 make kubeapps/kubeapps-apis
kind load docker-image kubeapps/kubeapps-apis:dev1 --name kubeapps-for-pinniped
kubectl set image deployment/kubeapps-internal-kubeappsapis -n kubeapps kubeappsapis=kubeapps/kubeapps-apis:dev1 --record
kubectl scale deploy/kubeapps-internal-kubeappsapis -n kubeapps --replicas=1
```
But having it there is neater!  Thanks!